pkg/acsengine/pki.go

```diff
@@ -68,6 +72,18 @@ func CreatePki(extraFQDNs []string, extraIPs []net.IP, clusterDomain string, caP
 
 	go func() {
 		var err error
+		etcdCertificate, etcdPrivateKey, err = createCertificate("etcd", caCertificate, caPrivateKey, true, extraFQDNs, extraIPs)
```

Comment:
This line is where you are adding ips and fqdns. We just want the ips if I'm not mistaken.